src/coders/byte-packets.js

```diff
@@ -1,9 +1,56 @@
+/**
+ * @typedef {function} PacketConstructor
+ */
+
+/**
+ * A packet of bytes represented with getter/setter properties for decoding and
+ * encoding values at given offsets mapped to names.
```

Comment:
This sentence is a little confusing. What is "mapped to names"? If it's the `values` maybe rephrase to:

```suggestion
  * encoding values mapped to names, located at given offsets.
```

Reply:
OR:

```suggestion
  * encoding values at given offsets as names.
```